app/Console/Commands/ForceDeleteTestRecords.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Post;
+use Rogue\Models\Signup;
+use Illuminate\Console\Command;
+
+class ForceDeleteTestRecords extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:forceDeleteTestRecords';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Force deletes signups and posts made in Runscope and Ghost Inspector tests';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        // Force delete all signups created by Runscope, identified by source.
+        $runscopeSignups = Signup::where('source', 'runscope')->orWhere('source', 'runscope-oauth')->get();
+        foreach ($runscopeSignups as $runscopeSignup) {
+            $runscopeSignup->forceDelete();
+        }
+
+        // Force delete all posts created by Runscope, identified by source.
+        $runscopePosts = Post::where('source', 'runscope')->orWhere('source', 'runscope-oauth')->get();
+        foreach ($runscopePosts as $runscopePost) {
+            $runscopePost->forceDelete();
+        }
+
+        $this->info('All Runscope and Ghost Inspector Signups deleted.');
```

Comment:
I think this log should be moved to line 61 after you delete the ghost inspector signups. I think you can also use plain `info('All Runscope and Ghost Inspector Signups deleted.');`

Reply:
oh shoot you're right! I meant to say `All Runscope Signups and Posts deleted` :) Thanks for catching!

Reply:
@katiecrane I tried doing just `info('All Runscope and Ghost Inspector Signups deleted.');` but that didn't output any message!

Reply:
hmmm that's weird, that seems to be how our other commands do it

Reply:
🤔